source/actions/random-desktop.js

```diff
@@ -33,32 +42,36 @@
    *  Otherwise randomly set desktop background
    */
   if (!savedImages.length) {
-    logger.warn(chalk.redBright(`\n${ERROR_CODES.ER04}: No existing images, try getting the images first`));
-    logger.info('Type get-lock-screen get-images');
-  } else {
-    /* Only pick landscape images */
-    const savedLandscapeImages = filterImages(savedImages, { orientation: ORIENTATION_LANDSCAPE });
-    const pick = `${currentSavePath.toString()}`
-      + `/${savedLandscapeImages[Math.floor(Math.random() * savedLandscapeImages.length)].name}`;
-
-    const result = await taskExecutor(
-      /* 3. Randomly set desktop background and announce */
-      setWallpaper(pick),
-      `Found ${savedImages.length} images. Picking a random wallpaper..`,
-      500,
+    logger.error(
+      'No existing images, try getting the images first.'
+      + `${chalk.white('\nType get-lock-screen get-images')}`,
+      {errorCode: ERROR_CODES.RUNTIME_ERROR_003},
     );
+    return printJsonOutput(logger);
+  }
+  /* Only pick landscape images */
+  const savedLandscapeImages = filterImages(savedImages, {orientation: ORIENTATION_LANDSCAPE});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required before '}'.](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5102014)